sdcm/tester.py

```diff
@@ -790,9 +794,11 @@ def run_gemini(self, cmd, duration=None):
 
         timeout = self.get_duration(duration)
         time.sleep(30)
+        test_node = random.choice(self.db_cluster.nodes)
+        oracle_node = random.choice(self.cs_db_cluster.nodes)
         return self.loaders.run_gemini_thread(cmd, timeout, self.outputdir,
-                                              test_node=self.db_cluster.nodes[0].private_ip_address,
-                                              oracle_node=self.cs_db_cluster.nodes[0].private_ip_address)
+                                              test_node=test_node.private_ip_address,
```

Comment:
do you need `private_ip_address`? or you can use `ip_address`?

Reply:
fixed